purchase_request_type/tests/test_purchase_request_type.py

```diff
@@ -74,6 +74,7 @@ def test_compute_purchase_request_type(self):
         self.assertEqual(request2.reduce_step, self.type2.reduce_step)
 
     def test_purchase_request_type_company_error(self):
+        self.type1.company_id = self.env.company
```

Comment:
This sounds weird if you need to assign the company explicitly. Isn't this revealing a problem?

Reply:
That's right, the data `tier.definition` has no `company_id` defined and this test is intended to check what happens if a tier.definition has company_id defined but, again, if you want this to work differently comment it and I will modify it.